### PR TITLE
docs: Updated employee.md (HR Module)

### PR DESCRIPTION
**Pull request by michellealva**
Updated the entire employee.md file with added explanation:

screenshot-erp com-8000-2019 06 21-14-58-59

### CODE REVIEW

**Comment by michellealva** [comment]
@prasadarr kindly review

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 1]
```diff
@@ -3,39 +3,72 @@
...
+
 
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-1.png">
 
-You can quickly add Weekly Off in the Holiday List as following.
+## 2. Features 
+
+Some of the additional features in the Holiday List doctype are as follows:
+
+### 1. Add Weekly Holidays
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]:
```suggestion
### 2.1 Add Weekly Holidays 
```
follow this in the following subsections also

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 2]
```diff
@@ -3,39 +3,72 @@
...
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-1.png">
 
-You can quickly add Weekly Off in the Holiday List as following.
+## 2. Features 
+
+Some of the additional features in the Holiday List doctype are as follows:
+
+### 1. Add Weekly Holidays 
+
+You can quickly add Weekly Offs in the Holiday List as following:
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]:
```suggestion
You can quickly add Weekly Offs in the Holiday List as follows:
```

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 3]
```diff
@@ -3,39 +3,72 @@
...
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-2.gif">
 
-After that, you can also add specific days (like festival holidays) manually.
+
+After that, you can also add specific days (like festival holidays) manually by clicking on the 'Add row' option in the Holidays table.
 
 <img class="screenshot" alt="Holiday List" src="{{docs_base_url}}/assets/img/human-resources/holiday-list-3.png">
 
+> Note: 
+Each time a new holiday is updated in the Holidays table, the Total Holidays field gets updated.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]: take this to the previous line

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 4]
```diff
@@ -3,39 +3,72 @@
...
+
+
+### 3. Holiday List in Employee 
+
+If you have created multiple Holiday List, select a specific Holiday List for an Employee in the respective master.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]:
```suggestion
If you have created multiple Holiday Lists, select a specific Holiday List for an Employee in the respective master.
```

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 5]
```diff
@@ -3,39 +3,72 @@
...
+
+When an Employee applies for Leave, the days mentioned in the Holiday List will not be counted, as they are holidays already. 
+
+
 
-### Holiday List in Employee
+> Note: If you have specified a Holiday List in the Employee master, then that Holiday List will be given more priority as compared to the default Holiday List of the company.
+You can form as many holiday lists as you wish. For example, if you have a factory, you can have one list for the factory workers and another list for office staff. You can manage between many lists by linking a Holiday List to the respective Employee.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]:
```suggestion
> You can form as many holiday lists as you wish. For example, if you have a factory, you can have one list for the factory workers and another list for office staff. You can manage between many lists by linking a Holiday List to the respective Employee.
```
take care how the formatting looks

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md [hunk 6]
```diff
@@ -3,39 +3,72 @@
...
 
-If you have created multiple Holiday List, then select specific Holiday List for an Employee in the respective master.
+You can also access [Company](/docs/user/manual/en/setting-up/company-setup), [Employee](/docs/user/manual/en/human-resources/employee), [Workstation](/docs/user/manual/en/manufacturing/workstation), [Service Level](/docs/user/manual/en/support/service-level), [Service Level Agreement](/docs/user/manual/en/support/service-level-agreement), [Leave Period](/docs/user/manual/en/human-resources/leave-period), and [Shift Type](/docs/user/manual/en/human-resources/shift-type) from the Holiday List page.
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]:
```suggestion
After saving, you can also access [Company](/docs/user/manual/en/setting-up/company-setup), [Employee](/docs/user/manual/en/human-resources/employee), [Workstation](/docs/user/manual/en/manufacturing/workstation), [Service Level](/docs/user/manual/en/support/service-level), [Service Level Agreement](/docs/user/manual/en/support/service-level-agreement), [Leave Period](/docs/user/manual/en/human-resources/leave-period), and [Shift Type](/docs/user/manual/en/human-resources/shift-type) from the Holiday List dashboard. 
```

File: erpnext_com/www/docs/user/manual/en/human-resources/hr-settings.md [hunk 7]
```diff
@@ -1,8 +1,23 @@
 <!-- add-breadcrumbs -->
+
 # HR Settings
 
-Global settings for HR related documents
+**HR Settings allow global settings for HR related documents.**
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/hr-settings.md]: skip this file, already have a PR for it

File: erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md [hunk 8]
```diff
@@ -1,8 +1,10 @@
 # Leave Type
 
+**Leave Type refers to types of leave allotted to an Employee by a company.** 
+
 > Human Resources > Leaves and Holiday > Leave Type > New Leave Type
```
Comment (prasadarr) [inline erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md]:
```suggestion
To access Leave Type, go to:
> Home > Human Resources > Leaves and Holiday > Leave Type
```
is this incomplete?

Comment (michellealva) [reply erpnext_com/www/docs/user/manual/en/human-resources/leave-type.md]: yes

File: erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md (hunk 5 above)
Comment (michellealva) [reply erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]: Even if you put > , it takes it as one block only.

Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/human-resources/holiday-list.md]: the point is to have a line break, not one big chunk of text, so line break, then >\

File: erpnext_com/www/docs/user/manual/en/human-resources/employee.md [hunk 9]
```diff
@@ -1,38 +1,118 @@
...
+
+### 2.8 Personal Details
+
+In this section, all the Personal Details of the Employee such as Family Background details like name and occupation of parent, spouse and children, Passport Details including date and place of issue, Health Details like height, weight, allergies, medical concerns, etc. are saved. 
+
+### 2.9 Educational Qualification
+
+The Educational Qualification details of the Employee can be filled in the Education table. To add a new entry, click on the 'Add Row' button. Details such as School/University, Qualification, Level and Year of Passing can be filled as shown below:
+
+<img class="screenshot" alt="Educational Qualification" src="{{docs_base_url}}/assets/img/human-resources/educational-qualification.png">
```
[the remark this answers is not in the document]
Comment (prasadarr) [reply erpnext_com/www/docs/user/manual/en/human-resources/employee.md]: image top is cut off here

File: erpnext_com/www/docs/user/manual/en/human-resources/employee.md [hunk 10]
```diff
@@ -1,38 +1,118 @@
...
+
+
+### 2.7 Contact Details
+
+In this section, the Employee's Contact information such as Mobile number, Current and Permanent Address, Personal and Company Email ID can be captured. In the Preferred Email ID field, either the the Company Email, Personal Email or User ID of the Employee can be selected.
+ 
+
+### 2.8 Personal Details
+
+In this section, all the Personal Details of the Employee such as Family Background details like name and occupation of parent, spouse and children, Passport Details including date and place of issue, Health Details like height, weight, allergies, medical concerns, etc. are saved.
```
[the remark this answers is not in the document]
Comment (michellealva) [reply erpnext_com/www/docs/user/manual/en/human-resources/employee.md]: remove comma after etc.

**Pull request closed without merging**